lib/server.js

```diff
@@ -728,6 +728,7 @@ WalletService.prototype._selectTxInputs = function(txp, cb) {
           if (!bitcoreError) {
             txp.inputPaths = _.pluck(txp.inputs, 'path');
             txp.fee = bitcoreTx.getFee();
+            $.checkState(txp.fee < 1e8, 'Fees are too high!');
```

Comment:
This should calculate fee per kb of the estimated size and warn if higher than 10mBTC per kb.